src/ui/YouTube/YouTubeThumbnail.ts

```diff
@@ -28,22 +33,28 @@ export class YouTubeThumbnail extends Component {
    * @componentOptions
    */
   static options: IYouTubeThumbnailOptions = {
+
     /**
-     * Specify the width that the thumbnail should have.
+     * Specifies the width (in pixels) of the YouTube thumbnail.
      *
-     * The default value is `200px`
+     * Default value is `200px`.
      */
```

Comment:
We would need to have a specific type for "Sizes", and then in the Typedoc project, have a way to recognize that type specifically and generate correct examples.

It would make a lot of sense to have a dedicated type for options that requires size. It would be helpful in the Interface Editor also.

If you are interested we can pass some time together and I can give you a crash course :)